### PR TITLE
Small fixes in the README file related to NVHPC

### PR DESCRIPTION
**Pull request by iomaganaris**

### CODE REVIEW

File: docs/CC2023/README.md [hunk 1]
```diff
@@ -87,7 +87,8 @@ generated containing the CPU results.
...
 ## Notes
 
-1. Acceleration results with `GCC` compiler might be better in the docker container than the paper
-   due to the newer OS we're using in the Dockerfile. Latest Ubuntu versions come with GLIBC 2.3x that
-   includes `libmvec` which provides vectorized implementations to the `GCC` compiler enabling the
-   vectorization of the kernels even without providing the `SVML` library to `GCC`.
+1. Acceleration results with `GCC` and `NVHPC` compilers might be better in the docker container than
+   the paper due to the newer OS we're using in the Dockerfile. Latest Ubuntu versions come with
+   GLIBC 2.3x that includes `libmvec` which provides vectorized implementations to the `GCC` and
+   `NVHPC` compiler (which is using as base compiler `GCC`) enabling the vectorization of the kernels
```
Comment (georgemitenkov) [inline docs/CC2023/README.md:93]: compiler --> compilers

Comment (georgemitenkov) [reply docs/CC2023/README.md:93]: which is using as base compiler `GCC` --> which is using `GCC` as the base compiler

**Review by georgemitenkov — APPROVED** [review]
Looks good, added some minor suggestions

File: docs/CC2023/README.md (hunk 1 above)
Comment (iomaganaris) [reply docs/CC2023/README.md:93]: Thanks for the corrections 👍

**Pull request merged**